wandb/proto/wandb_internal.proto

```diff
@@ -424,6 +424,7 @@ message ArtifactManifestEntry {
   string birth_artifact_id = 7;
   // Whether to avoid copying/moving files to the cache while uploading.
   bool skip_cache = 8;
+  string policy = 9;
```

Comment:
is the plan to have more policies? if it is just mutable vs immutable bool would suffice. 
Also what do you think of making it an enum or something more typed? i assume it can't be any string right?

Reply:
We plan on eventually adding a `delete` policy too, which instead of copying the original files to staging, will move them. 

Made it an enum instead!